client/management/v3/zz_generated_rancher_kubernetes_engine_config.go

```diff
@@ -32,6 +33,7 @@ type RancherKubernetesEngineConfig struct {
 	BastionHost         *BastionHost       `json:"bastionHost,omitempty" yaml:"bastionHost,omitempty"`
 	CloudProvider       *CloudProvider     `json:"cloudProvider,omitempty" yaml:"cloudProvider,omitempty"`
 	ClusterName         string             `json:"clusterName,omitempty" yaml:"clusterName,omitempty"`
+	DinD                bool               `json:"dind,omitempty" yaml:"dind,omitempty"`
```

Comment:
what is the reason behind adding this as an option in rkeconfig, what is the use case of using it without the cli?